generators/app/templates/gulp/tasks/vendor.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+
+var bundleVendorTask = function (gulp, plugins, config) {
+  var fileListPath = path.join(config.src.base, 'scripts/vendor.json');
+  var getFileList = function () {
+    return JSON.parse( fs.readFileSync(fileListPath) );
+  };
+
+  gulp.task('vendor-clean', function(done) {
+    return plugins.del([ path.join(config.dest.base, 'scripts/vendor.js') ], done);
```

Comment:
What would be a more straightforward approach here? Couldn't see anything simpler. Possibly because lack of experience with Gulp. Your assistance would be welcome here. 👍